src/Task.h

```diff
@@ -1101,6 +1108,9 @@ class Task {
   // True when 'registers' has changes that haven't been flushed back to the
   // task yet.
   bool registers_dirty;
+  // True when changes to the original syscallno in 'registers' have not been
+  // flushed back to the task yet.
```

Comment:
Add "Some architectures (e.g. AArch64) require a separate ptrace call for this."

Reply:
Will do.